lib/oxrdf/Cargo.toml

```diff
@@ -16,13 +16,16 @@ rust-version.workspace = true
 [features]
 default = []
 rdf-star = []
+serde = ["dep:serde", "dep:serde_json"]
 
 [dependencies]
 oxilangtag.workspace = true
 oxiri.workspace = true
 oxsdatatypes = { workspace = true, optional = true }
 rand.workspace = true
 thiserror.workspace = true
+serde = { version = "1.0.217", optional = true, features = ["derive"] }
+serde_json = { version = "1.0.138", optional = true }
```

Comment:
It would be great to avoid this dependency, at least on the non-test path

Reply:
`serde_json` is indeed only used by tests; so it could be a `dev-dependency`. However, cargo does not support optional dev-dependencies (https://github.com/rust-lang/cargo/issues/1596). So the options are:
 - Make it a non-optional dev-dependency
 - Make it an optional dependency
 - Create a new package for testing purposes only which uses `serde_json`
 
 A new package might be the way to go, this is also required [should tests for the zkvm environment need to be added](https://github.com/rust-lang/rust/issues/135376#issuecomment-2657851915).